src/newt/db/search.py

```diff
@@ -173,6 +173,10 @@ def create_text_index(conn, fname, D, C=None, B=None, A=None):
         conn.commit()
     finally:
         try:
+            cursor.close()
```

Comment:
So the leak check code that I use in RelStorage complains when both cursors and connections are left open to be GCd. That's how I found this.